app/code/core/Mage/Sales/Model/Quote/Item.php

```diff
@@ -516,12 +516,16 @@ public function compare($item)
                 // dispose of some options params, that can cramp comparing of arrays
                 if (is_string($itemOptionValue) && is_string($optionValue)) {
                     try {
-                        /** @var Unserialize_Parser $parser */
+                        /**
+                         * @var Mage_Core_Helper_UnserializeArray $parser
+                         * @var Mage_Core_Helper_String $stringHelper
+                         */
```

Comment:
I think it's unnecessary
```suggestion
```

Reply:
I think IDE would prefer a type hint for the helpers

Reply:
OK, it's harmless. FYI, VSCode + Intelephense has no problem with helpers. I am now using cursor, no issue as well,